docs/process/using-the-staging-environment.md

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
I opted to split out using staging generically from using staging specifically as it relates to testing Sync.